apps/website/app/ui/page.tsx

```diff
@@ -649,6 +651,11 @@ export default function UI() {
                 <DialogHeader>
                   <DialogTitle>Hi I'm a Modal title</DialogTitle>
                 </DialogHeader>
+                <VisuallyHidden asChild>
+                  <DialogDescription className="px-4">
+                    Here a really long text will be displayed.
+                  </DialogDescription>
+                </VisuallyHidden>
```

Comment:
Shouldn't we add `VisuallyHidden` to `DialogDescription` and `DialogTitle`??

Reply:
I mean, add it to the component it self. Cause we are using it everywhere and looks like every `DialogTitle` need a `VisuallyHidden` component. So probably we should wrap the component with `VisuallyHidden` on Dialog file. What do you think?

Reply:
Mmm, I'm not sure if I'm understanding correctly. 

Do you mean to modify how we export `DialogTitle` to be wrapped with `VisuallyHidden` by default? 
If that's the case then we would be hiding all our already visible DialogTitles in Presagio for example:
[Preagio's ConfirmTrade component](https://github.com/SwaprHQ/presagio/blob/7536ca6e60416fa6c8bc226d8597c0609717ef2b/app/components/ConfirmTrade.tsx#L230)
[Presagio's EmbedMarket component](https://github.com/SwaprHQ/presagio/blob/7536ca6e60416fa6c8bc226d8597c0609717ef2b/app/components/EmbedMarketModal.tsx#L99)

We don't need to render the title in the TransactionModal or similar components.
I would:
* Export it as-is from our lib
* Let the clients from this package install the `VisuallyHidden` component and decide whether use it, or implement another solution to hide the component
* Maybe do some conditional rendering like:
```tsx
hidden ? (
  <VisuallyHidden asChild>
    <DialogDescription {...props}>
      {description}
    </DialogDescription>
  </VisuallyHidden>
) : (
  <DialogDescription {...props}>
   {description}
  </DialogDescription>
)
```

Reply:
I now understand how it works, let's not to that! 😄 

But we can export `VisuallyHidden` from Dialog file maybe? And we don't need to had a new lib to presagio package.json.